about.html

```diff
@@ -0,0 +1,137 @@
+          <li><a class="text-decoration-none mobile-menu-item fw-semibold display-3" href="about.html#cf-campaign">CF Campaign</a></li>
+        </ul>
+      </div>
+      <div><a href="index.html"><img class="pt-1 ms-5" src="./assets/mobile/cflogo.png" width="40" height="40" alt="Concert Festival Logo"></a></div>
+    </div>
+  </nav>
+</div>
+  <main>
+    <section class="about-main-page pt-5">
+      <div class="container text-md-center pt-4 pb-4">
+        <h1 class="display-4 fw-bold welcome">CONCERT FESTIVAL</h1>
+        <p class="display-3 fw-bold welcome">2023</p>
+      </div>
+
+      <div class="mx-auto about-description bg-light">
+        <p class="p-3">The Concert Festival is one of World's most
+          anticipated cultural events . Rehearsals are open to the public
+          and concerts feature both classical masterworks and contemporary
+          pieces performed by world-renowned musicians. The Festival has
+          played to full houses of chamber music enthusiasts from around
+          the world and has received brilliant reviews every year.
+        </p>
+      </div>
+      <div class="contact text-center py-3">
+        <span class="contact lead">Please contact us per Email for any further questions</span><br>
+        <span class="contact lead">about Concert Festival 2021.</span><br>
+        <span class="contact lead fw-semibold text-decoration-underline">info@cfestival.org</span>
+      </div>
+    </section>
+    <section id="cf-campaign">
+      <div class="text-center">
+        <h2>Concert Festival 2023 Logo</h2>
+        <span class="orange-bar mx-auto"></span>
+      </div>
+      <div class="text-center my-4">
+        <span class="text-center">The logo of Concert Festival 2023 was decided</span><br>
+        <span class="text-center">through the logo competition held from 1.Sept to 3.Oct</span>
+        <div class="cf-logo my-2">
+          <img src="./assets/mobile/cf-logo-white.jpg" alt="Concert Festival Logo" width="300" height="160">
+        </div>
+        <hr>
+      </div>
+
+      <div class="text-center">
+        <h2>See The Past Concert Festival</h2>
+        <span class="orange-bar mx-auto"></span>
+        <div class="text-center ">
+          <span>Take a look at the last two Concert Festival which took place</span><br/>
+          <span>in Moscow and Cairo</span>
+        </div>
+        <div class="past-events d-md-flex my-md-3 mx-auto">
+          <div class="moscow-event text-light">
+            <h2 class="display-3 fw-semibold">2022</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Moscow</p>
+          </div>
+          <div class="text-light cairo-event">
+            <h2 class="display-3 fw-semibold">2021</h2>
+            <p class="lead fw-semibold">Concert Festival 2022 in Cairo</p>
+          </div>
+        </div>
+      </div>
+    </section>
```

Comment:
- [ ] Kindly, as is the case on the provided template, You may consider giving your image the orange background 
- [ ] Also you may consider adding the footer on this page as well as is the case in the provided template.

Your design :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203995632-1f5eacb2-1000-4ad7-bfbc-e04579809486.png)

The template :point_down: 
![image](https://user-images.githubusercontent.com/97100091/203995764-ef314ac2-c106-4eeb-b728-cd763818ab9b.png)